tools/validate/validate.cpp

```diff
@@ -54,7 +54,7 @@ class BinaryFile {
 
 enum class TabPolicy : bool { Forbidden, Allowed };
 
-void scan_file(const filesystem::path& filepath, const TabPolicy tab_policy, vector<unsigned char>& buffer) {
+void scan_file(const std::filesystem::path& filepath, const TabPolicy tab_policy, vector<unsigned char>& buffer) {

```

Comment:
you need also change `vector` to `std::vector`
and `array` to `std::array`
and also the file contains: `L".git"sv` and similar.
Actually I don't remember how to use `sv` without `using namespace`
I wrote `using namespace std::literals;` and then validate.cpp is built succesfully on my PC.

Reply:
would you please check it again? where i did go wrong?